docs/concepts/multi-chain-proposals.md

```diff
@@ -0,0 +1,693 @@
+---
+id: multi-chain-proposals
+title: Multi-Chain Proposals
+---
+
+This is a living document which represents the current process guidelines for developing and advancing multi-chain Uniswap Governance Proposals. It was last updated Novenber 2024.
+
+## Introduction
```

Comment:
can we combine Introduction into Overview? I think there's also opportunity to shorten these sections, we can take another pass together after other feedback is addressed.

Reply:
Done, basically combine them.. interested in your thoughts on how to shorten them: 1e62f91c018922509c01f0a75e5fe6b6d8153f53